addons/l10n_in_ewaybill_stock/models/l10n_in_ewaybill.py

```diff
@@ -240,14 +245,30 @@ def cancel_ewaybill(self):
         }
 
     def reset_to_pending(self):
-        if self.state != 'cancel':
-            raise UserError(_("Only Cancelled E-waybill can be resent."))
+        self.ensure_one()
```

Comment:
Are you sure it can not be in handy to do things in batch?  (otherwise ok)

Reply:
As of now we don't have a list view for it. (I guess refactor might have it 😬 ). Anyways it's easy to change it in future if needed. since the current code doesn't belongs to `for records in self:` we are directly checking the `self.state` raising `self.ensure_one()` is much better for now